### PR TITLE
Changing cfg files for fedora 30 to use tpl file

### PR DESCRIPTION
**Pull request by schlupov**

### CODE REVIEW

File: mock-core-configs/etc/mock/fedora-30-aarch64.cfg [hunk 1]
```diff
@@ -1,168 +1,5 @@
...
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/$releasever/Modular/SRPMS/
-metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-source-f$releasever&arch=$basearch
-enabled=0
-repo_gpgcheck=0
-type=rpm
-gpgcheck=1
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
-skip_if_unavailable=False
-"""
+config_opts['baseurl'] = http://armpkgs.fedoraproject.org/repos/f30-build/latest/aarch64/
```
Comment (praiskup) [inline mock-core-configs/etc/mock/fedora-30-aarch64.cfg]: This url doesn't work for me.

File: mock-core-configs/etc/mock/fedora-30-armhfp.cfg [hunk 2]
```diff
@@ -1,174 +1,5 @@
...
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/$releasever/Modular/SRPMS/
-metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-source-f$releasever&arch=$basearch
-enabled=0
-repo_gpgcheck=0
-type=rpm
-gpgcheck=1
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
-skip_if_unavailable=False
-"""
+config_opts['baseurl'] = https://kojipkgs.fedoraproject.org/repos/f30-build/latest/armhfp/
```
Comment (praiskup) [inline mock-core-configs/etc/mock/fedora-30-armhfp.cfg]: You probably want to use $basearch here.

File: mock-core-configs/etc/mock/fedora-30.tpl [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+[fedora-debuginfo]
+name=fedora-debuginfo
+metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-debug-$releasever&arch=$basearch
+enabled=0
+gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-30-primary
+gpgcheck=1
+skip_if_unavailable=False
+{% if target_arch == 'armv7hl' %}
+    failovermethod=priority
+{% endif %}
```
Comment (praiskup) [inline mock-core-configs/etc/mock/fedora-30.tpl]: Can we avoid this condition (isn't it actually bug in the old configuration)?

**Comment by praiskup** [comment]
If you **only** moved one file into the template in first commit, and you made the rest of the changes in another commit it would be much easier to read and review (and you would protect the git from loosing the history, aka git blame functionality).

**Comment by praiskup** [comment]
Also, what if we moved the templates into `/etc/mock/templates`?

File: mock-core-configs/etc/mock/fedora-30-aarch64.cfg (hunk 1 above)
Comment (praiskup) [reply mock-core-configs/etc/mock/fedora-30-aarch64.cfg]: You want to use https://kojipkgs.fedoraproject.org/repos/f30-build/latest/$basearch/ in all files, so you can actually avoid this.

**Comment by nkadel** [comment]
@praiskup I think that moving the templates to /etc/mock/templates/ makes complete sense. I'd encourage it as a separate pull request, because it's a distinct issue and will touch a stack of config files.

**Pull request merged**